core/src/epicli/data/common/ansible/playbooks/roles/kafka/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 specification:
```

Comment:
`specification` can be defined by user and used in `configuration` part, for role defaults it's not so and I think it should be changed here.

Reply:
Could you elaborate more? And show some example on ` like we started to do for other roles.`?

Reply:
Here is the [thread](https://github.com/epiphany-platform/epiphany/pull/1684#discussion_r491903720) to follow. As I remember all necessary changes were done there, so it can be used as an example.

Reply:
Modified kafka defaults to have values that will not be editable by the user.